<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>751</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="13" column="2">
      <widget class="QPushButton" name="STL_Sort">
       <property name="text">
        <string>Sort using STL Sort</string>
       </property>
      </widget>
     </item>
     <item row="10" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="2">
      <widget class="QListWidget" name="listWidget"/>
     </item>
     <item row="10" column="1">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="12" column="0">
      <widget class="QLineEdit" name="sizeEDIT">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Input Dataset Size</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="method">
         <property name="text">
          <string>Choose your search method </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BinarySearch">
         <property name="text">
          <string>Binary Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="NormalSearch">
         <property name="text">
          <string>Normal Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="STLSearch">
         <property name="text">
          <string>STL find</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="12" column="2">
      <widget class="QPushButton" name="B_Sort">
       <property name="text">
        <string>Sort using Bubble Sort</string>
       </property>
      </widget>
     </item>
     <item row="12" column="4">
      <widget class="QPushButton" name="find_it">
       <property name="text">
        <string>Find it </string>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QPushButton" name="Q_Sort">
       <property name="text">
        <string>Sort using Quick Sort</string>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QPushButton" name="M_Sort">
       <property name="text">
        <string>Sort using Merge Sort</string>
       </property>
      </widget>
     </item>
     <item row="9" column="4">
      <widget class="QLabel" name="DisplaySize">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="10" column="4">
      <widget class="QLabel" name="sortingType">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="arrayStatus">
       <property name="text">
        <string>    Unsorted Array </string>
       </property>
      </widget>
     </item>
     <item row="11" column="4">
      <widget class="QLineEdit" name="num_find">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QPushButton" name="Set_Generator">
       <property name="text">
        <string>Generate Dataset</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="4">
      <widget class="QLabel" name="searchMethod">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
